From b3bc639206ce6cd3487ba4fc5de8e79dd8b0f8a5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Gast=C3=B3n=20Kleiman?= <gaston@mesosphere.io>
Date: Mon, 21 Aug 2017 15:28:39 -0700
Subject: [PATCH 07/13] Raised the logging level of some check and health check
 messages.

Some users pointed out that always logging the result of checks and
health checks makes it easier to debug problems.

Review: https://reviews.apache.org/r/61791/
---
 src/checks/checker_process.cpp | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/checks/checker_process.cpp b/src/checks/checker_process.cpp
index 47d864803..204510660 100644
--- a/src/checks/checker_process.cpp
+++ b/src/checks/checker_process.cpp
@@ -811,7 +811,7 @@ void CheckerProcess::processCommandCheckResult(
   // see MESOS-7242.
   if (future.isReady() && WIFEXITED(future.get())) {
     const int exitCode = WEXITSTATUS(future.get());
-    VLOG(1) << name << " for task '" << taskId << "' returned: " << exitCode;
+    LOG(INFO) << name << " for task '" << taskId << "' returned: " << exitCode;
 
     CheckStatusInfo checkStatusInfo;
     checkStatusInfo.set_type(check.type());
@@ -972,8 +972,8 @@ void CheckerProcess::processHttpCheckResult(
   Result<CheckStatusInfo> result = None();
 
   if (future.isReady()) {
-    VLOG(1) << name << " for task '" << taskId << "'"
-            << " returned: " << future.get();
+    LOG(INFO) << name << " for task '" << taskId << "'"
+              << " returned: " << future.get();
 
     CheckStatusInfo checkStatusInfo;
     checkStatusInfo.set_type(check.type());
@@ -1113,8 +1113,8 @@ void CheckerProcess::processTcpCheckResult(
   Result<CheckStatusInfo> result = None();
 
   if (future.isReady()) {
-    VLOG(1) << name << " for task '" << taskId << "'"
-            << " returned: " << future.get();
+    LOG(INFO) << name << " for task '" << taskId << "'"
+              << " returned: " << future.get();
 
     CheckStatusInfo checkStatusInfo;
     checkStatusInfo.set_type(check.type());
-- 
2.17.0

